pints/_log_priors.py

```diff
@@ -104,22 +121,21 @@ def __init__(self, mean, variance):
         # Parse input arguments
         self._mean = float(mean)
         self._sigma = float(variance)
-        self._dimension = 1
 
         # Cache constants
         self._offset = 1 / np.sqrt(2 * np.pi * self._sigma ** 2)
         self._factor = 1 / (2 * self._sigma ** 2)
 
+    def __call__(self, x):
+        return self._offset - self._factor * (x[0] - self._mean)**2
+
     def dimension(self):
         """ See :meth:`LogPrior.dimension()`. """
         return 1
 
-    def __call__(self, x):
-        return self._offset - self._factor * (x[0] - self._mean)**2
-
     def sample(self, n=1):
         """ See :meth:`LogPrior.sample()`. """
-        return np.random.normal(loc=self._mean, scale=self._sigma, size=n)
+        return np.random.normal(self._mean, self._sigma, size=(n, 1))
```

Comment:
Just not sure, if we sample from a **log** normal, would that be the same as sampling from a normal distribution? Because it sounds more like "we sample from a **log** normal surface" rather than from a normal distribution.

Reply:
Same thing for the multivariate normal distribution.

Reply:
It would be different! In that case we'd call it a LogNormalLogPrior! This is the LogPrior that you get when you use a Normal distribution, that's why it's a NormalLogPrior!